.changeset/beige-ways-type.md

```diff
@@ -0,0 +1,6 @@
+---
+'@suspensive/react': patch
+'@suspensive/react-query': patch
+---
```

Comment:
We can check how changeset action will work. when we merge this Pull Request
changeset/action will make Pull Request containing CHANGELOG.md